src/promptflow/promptflow/executor/flow_executor.py

```diff
@@ -1301,12 +1310,16 @@ def execute_flow(
     :type enable_stream_output: Optional[bool]
     :param run_id: Run id will be set in operation context and used for session.
     :type run_id: Optional[str]
+    :param enable_async_execution: If True, AsyncNodeScheduler will be used by default to execute node.
```

Comment:
Maybe we can still use env variable to control?

Reply:
Updated.